client/my-sites/earn/home.tsx

```diff
@@ -492,7 +493,12 @@ const Home = () => {
 					<PromoSection promos={ [ getPlaceholderPromoCard(), getPlaceholderPromoCard() ] } />
 				</div>
 			) }
-			{ ! isLoading && <PromoSection { ...promos } /> }
+			{ ! isLoading && (
+				<>
+					<StatsSection />
+					<PromoSection { ...promos } />
+				</>
```

Comment:
Not currently loading any placeholder for this beyond what was there already for promo card. Could add that in follow up PR.

Reply:
Yeah, we can go through all the loaders separately little later 👍